oci/oci.go

```diff
@@ -591,7 +592,22 @@ func (r *Runtime) UpdateStatus(c *Container) error {
 
 	if c.state.Status == ContainerStateStopped {
 		exitFilePath := filepath.Join(r.containerExitsDir, c.id)
-		fi, err := os.Stat(exitFilePath)
+		var fi os.FileInfo
+		err = kwait.ExponentialBackoff(
+			kwait.Backoff{
+				Duration: 500 * time.Millisecond,
```

Comment:
One thing we need to check is whether this time is within any timeouts that kubelet enforces on status checks.

Reply:
is kube sending a timeout with a status call? (I'll check), this should be around 5 seconds if I remember the math I did yesterday lol

Reply:
No it isn't sending any. Just whatever is hard coded in the kubelet if any.

Reply:
Timeout in kube is by default https://github.com/kubernetes/kubernetes/blob/v1.7.4/pkg/apis/componentconfig/v1alpha1/defaults.go#L233 (we usually use 15m instead)

Reply:
okay, we should be good then. We can adjust this later if needed.